cmd/multiFaToChain/multiFaToChain.go

```diff
@@ -164,16 +185,22 @@ func queryState(records []fasta.Fasta, index int) (State, bool) {
 
 func usage() {
 	fmt.Print(
-		"multiFaToChain - Convert a pairwise multiFa format alignment to a chain file. First species is the target by default.\n" +
+		"multiFaToChain - Convert a multiFa format alignment to a chain file. First species is the target by default.\n" +
+			"Arguments:\n" +
+			"\tinFile: a multiFa format alignment, where the first sequence defines the coordinate system.\n" +
+			"\ttName: specifies the name of the target sequence (tName field) in the output chain. This will be a chromosome, construct, or plasmid name.\n" +
+			"\tqName: specifies the name of the query chromosome (qName field) in the output chain. This will be a chromosome, construct, or plasmid name.\n" +
+			"\toutFile: a UCSC chain format file, specifying a coordinate mapping between the target and query sequence.\n" +
 			"Usage:\n" +
-			" multiFaToChain input.fa tName qName output.chain\n" +
+			"\tmultiFaToChain input.fa tName qName output.chain\n" +
 			"options:\n")
 	flag.PrintDefaults()
 }
 
 func main() {
 	var expectedNumArgs int = 4
 	var swapTandQ *bool = flag.Bool("swapTandQ", false, "Swap the target and query in the output chain file.")
+	var querySeqName *string = flag.String("querySeqName", "", "Specify which sequence in the multiFa to use as the query for chain conversion.")
```

Comment:
Some food for thought, but what do you think about the style where defining these flags in shorthand for programs written in cmd (package main)

For example, :I personally would omit the  declaration `var querySeqName *string()` and use `querySeqName :=`(colon equals) because shortening these flags makes it easier to read/understand.

For L203 I would've liked something like::
```
querySeqName := flag.String("querySeqName", "", "Sequence in multifa for chain conversion")
```
However, these are just my personal style preferences, I just wanted you to hear me out.